quarto/01-QC-DMT01.qmd

```diff
@@ -0,0 +1,161 @@
+---
+title: "QCing Demographics and Baseline Characteristics (DMT01)"
+editor: source
+format: 
+  html:
+    page-layout: full
+    code-fold: true
+    code-summary: "Show the code"
+    code-overflow: scroll 
+    df-print: kable
+---
+
+## Demographics and Baseline Characteristics (DMT01) QC Workflow
+
+#### 1. Generate a table using {chevron}
+
+```{r}
+#| message: false
+#| code-summary: "Show the code"
+library(chevron)
+
+# Create a table using the chevron package 
+tlg_dmt01 <- run(dmt01, syn_data)
```

Comment:
You can specify AGE instead of AAGE

Reply:
Here also the vars are renamed into Male and Female. You can take that out if you wish